domains/dem0n.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "probablydem0n",
+        "discord": 935272776586129419
```

Comment:
```suggestion
        "discord": "935272776586129419"
```

Reply:
sorry for the confusion, the numbers are my discord id incase I change my username (have done many times before) the discord id will not change and will always link to my latest username, so you can contact me as for telling about myself, do you want me to add that as a string in the json file or on the website instead? I was planning on doing that on the website.

Reply:
No, what i mant for yourself is, the website.

Reply:
Ah ok